### DISCUSSION TITLE
Rename sensor state class `TOTAL_INCREASING`

### DISCUSSION THREAD

**Discussion by emontnemery** [home-assistant/architecture#761] [Polls]
Rename sensor state class `TOTAL_INCREASING`

Background:
The sensor state class "total_increasing" should be used for sensors representing an amount which is periodically reset to 0 according to some cycle, e.g. an amount of consumed gas starting from 0 the 1st of every month.

Home Assistant's statistics does not need to know about the cycle, e.g. monthly or daily; it will interpret a decreasing value as the beginning of a new cycle and calculate a grand total of the consumption for all cycles which can only increase.

The name of the state class is poorly chosen and has caused a lot of confusion, because it reflects how Home Assistant core *converts the value when calculating statistics*, not what the sensor value represents - a sensor which represents a total and which will only increase should probably use state class `TOTAL`, not state class `TOTAL_INCREASING`.

Hence, we should try to find a name which better reflects the properties of the source: A total which periodically resets according to some unspecified cycle.

This PR https://github.com/home-assistant/core/pull/69902 suggests to rename `TOTAL_INCREASING` to `TOTAL_AUTO_CYCLE`. Several other proposals were raised in that PR, let's vote on them here 👍

**Comment by emontnemery** [comment]
Note: I'm not super happy with the single choice format of the poll, but it seems that's the only allowed option. Also, only 8 choices are allowed.

**Comment by emontnemery** [comment]
Context for the options:
- TOTAL_AUTO_CYCLE
  This was the original proposal in https://github.com/home-assistant/core/pull/69902
- TOTAL_AUTO_CYCLE_RESET
  https://github.com/home-assistant/core/pull/69902#issuecomment-1097228215
- TOTAL_AUTO_RESET
   https://github.com/home-assistant/core/pull/69902#issuecomment-1096785370
- TOTAL_INCREASING What we use today
- TOTAL_RESETTING
   https://github.com/home-assistant/core/pull/69902#issuecomment-1096663844
- TOTAL_SOURCE_CAN_RESET Clarify that the source may reset
   https://github.com/home-assistant/core/pull/69902#issuecomment-1097237669
- TOTAL_THAT_CAN_RESET
  https://github.com/home-assistant/core/pull/69902#issuecomment-1097237669
- WRAPPING
   https://github.com/home-assistant/core/pull/69902#issuecomment-1096831824

**Comment by frenck** [comment]
The results are pretty clear, for now, we'll stick with what we have.

Going to close this proposal for now.

../Frenck
